libs/dh/shared/feature-flags/.eslintrc.json

```diff
@@ -0,0 +1,36 @@
+{
+  "extends": ["../../../../.eslintrc.json"],
+  "ignorePatterns": ["!**/*"],
+  "overrides": [
+    {
+      "files": ["*.ts"],
+      "extends": [
+        "plugin:@nrwl/nx/angular",
+        "plugin:@angular-eslint/template/process-inline-templates"
+      ],
+      "rules": {
```

Comment:
suggestion: what do you think of adding `"no-dupe-keys": ["error"],` here in order to raise a lint error in case a duplicate feature flag is used?

Reply:
Nice one!